File: gvpr/Java/GvprLexerBase.java
import org.antlr.v4.runtime.*;

abstract class GvprLexerBase extends Lexer
{
    protected GvprLexerBase(CharStream input)
    {
        super(input);
    }

    protected boolean IsColumnZero()
    {
	return this.getCharPositionInLine() == 1;
    }
}

File: gvpr/Go/gvpr_lexer_base.go
package parser

import (
    "github.com/antlr4-go/antlr/v4"
)

type GvprLexerBase struct {
    *antlr.BaseLexer
}

func (l *GvprLexerBase) IsColumnZero() bool {
    return l.GetCharPositionInLine() == 1
}
